apps/dog-food/Main.hs

```diff
@@ -219,49 +218,4 @@ receiveFromAgent t c = forever . atomically $ do
       Disconnected _ -> set Nothing
```

Comment:
not in scope of this pr, but I still don't think it will be great to change active contact on each received message once you have more than one active conversation. Maybe we could have a separate command for that - e.g. "switchActiveContactToLatestSender" (wording could differ :), that would've been triggered by some specific key, say `Tab`. Let's consider it in future scope maybe, at least discuss it.. I just can see it being very annoying

Reply:
let's park this conversation till will have this problem - then it will be easier to understand what is better.